muxers/mplex/benches/split_send_size.rs

```diff
@@ -132,24 +130,16 @@ fn run(transport: &mut BenchTransport, payload: &Vec<u8>, listen_addr: &Multiadd
     // Spawn and block on the sender, i.e. until all data is sent.
     task::block_on(async move {
         let addr = addr_receiver.await.unwrap();
-        let (_peer, conn) = transport.dial(addr).unwrap().await.unwrap();
-        let mut handle = conn.open_outbound();
-        let mut stream = poll_fn(|cx| conn.poll_outbound(cx, &mut handle))
+        let (_peer, mut conn) = transport.dial(addr).unwrap().await.unwrap();
+        conn.open_outbound();
+        let (mut stream, _) = poll_fn(|cx| conn.poll(cx))
             .await
+            .unwrap()
+            .into_outbound_substream()
             .unwrap();
-        let mut off = 0;
-        loop {
-            let n = poll_fn(|cx| conn.write_substream(cx, &mut stream, &payload[off..]))
-                .await
-                .unwrap();
-            off += n;
-            if off == payload.len() {
-                poll_fn(|cx| conn.flush_substream(cx, &mut stream))
-                    .await
-                    .unwrap();
-                return;
-            }
-        }
```

Comment:
@mxinden This is also differently implemented in the benchmark now but I can't see how this would impact performance much.